doc/python/bubble-maps.md

```diff
@@ -156,52 +148,42 @@ fig['data'][0].update(mode='markers+text', textposition='bottom center',
 
 # Inset
 fig.add_trace(go.Choropleth(
-        locationmode = 'country names',
-        locations = df_sept['Country'],
-        z = df_sept['Value'],
-        text = df_sept['Country'],
-        colorscale = [[0,'rgb(0, 0, 0)'],[1,'rgb(0, 0, 0)']],
-        autocolorscale = False,
-        showscale = False,
-        geo = 'geo2'
-    ))
-fig.add_trace(go.Scattergeo(
-        lon = [21.0936],
-        lat = [7.1881],
-        text = ['Africa'],
-        mode = 'text',
-        showlegend = False,
-        geo = 'geo2'
-    ))
-
+        locationmode='country names',
+        locations=df_sept['Country'],
+        z=df_sept['Value'],
+        text=df_sept['Country'],
+        colorscale=[[0,'rgb(0, 0, 0)'],[1,'rgb(0, 0, 0)']],
+        autocolorscale=False,
+        showscale=False,
+        geo='geo2'))
```

Comment:
all these trailing parentheses should go back to being on their own line please

Reply:
I was hoping that this PR would be a simple "remove all `go.layout.whatever` but it actually contains a lot of hard-to-review/hard-to-undo changes like this, so is it possible to please break this up into multiple easier-to-review PRs? like one with just the "`go.layout.whatever` to `dict`" change and another with the  replacement of ` = ` to `=` etc?

Reply:
Ok, to make it less painful I created https://github.com/plotly/plotly.py/pull/2171 for removal of go.layout.* and will take care of formatting later